src/Microsoft.VisualStudio.LanguageServices.Razor/VsSolutionUpdatesProjectSnapshotChangeTrigger.cs

```diff
@@ -22,6 +23,27 @@ internal class VsSolutionUpdatesProjectSnapshotChangeTrigger : ProjectSnapshotCh
         [ImportingConstructor]
         public VsSolutionUpdatesProjectSnapshotChangeTrigger(
```

Comment:
`ProjectSnapshotChangeTrigger` is internal and we should make it an `ILanguageService` but I am not sure what's the equivalent of `ImportMany` in roslyn languageservices world. I'll add it to my [checklist](https://github.com/aspnet/Razor/issues/1799#issue-279962189) and figure this out.

Reply:
Discussed with @rynowak. Roslyn doesn't support getting services of `IEnumerable`. Going to leave `ProjectSnapshotChangeTrigger` as is.